CONTRIBUTING.md

```diff
@@ -2,18 +2,42 @@
 
 Operator Lifecycle Manager is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into operator-lifecycle-manager.
 
-# Email and Chat
+## Communication
 
-- Email: [operator-framework][operator_framework]  
+- Slack: [#olm-dev on the Kubernetes Slack](https://kubernetes.slack.com/archives/C0181L6JYQ2)
+- Google Group: [OLM Dev Google Group](https://groups.google.com/u/1/g/operator-framework-olm-dev)
+- Mailing List: [operator-framework-olm-dev@googlegroups.com](operator-framework-olm-dev@googlegroups.com)
+- Scheduled upstream meetings can be found at the [operator-framework/community GitHub Repository][community]
+
+> Note: By joining the Google Group you will automatically be added to the mailing list.
 
 ## Getting started
 
-- Fork the repository on GitHub
+- Fork and clone the repository on GitHub
```

Comment:
I think it would be helpful to describe this in more detail. Since the project is go based, your fork should be in the right directory structure and not in your personal GitHub directory structure.
```suggestion
- Fork and clone the repository on GitHub
  1. Fork the operator-lifecycle-manager repo (see [GitHub doc](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo))
  1. Clone your personal GitHub fork (prefer using SSH rather than HTTPS)

     ```bash
     mkdir -p $GOPATH/src/github.com/operator-framework
     cd $GOPATH/src/github.com/operator-framework
     git clone git@github.com:<YOUR PERSONAL ORG>/operator-lifecycle-manager.git
     cd operator-lifecycle-manager
     ```
   1. Add the upstream repo so you can keep your fork in sync
     
      ```bash
      git remote add upstream git@github.com:operator-framework/operator-lifecycle-manager.git
      ```

   1. Eventually you'll want to keep your master branch in sync with the upstream repo using commands like `git rebase upstream/master`
```

Reply:
Users shouldn't need their clone to be under GOPATH anymore (we explicitly set `GO111MODULE=on` in the Makefile), but it'd be worthwhile to tell users to install [at least Go 1.11](https://blog.golang.org/modules2019) for module support.

Reply:
I'll call this out in the next update.

Reply:
Addressed.

Reply:
See https://github.com/operator-framework/operator-lifecycle-manager/pull/1978#issuecomment-770093499